packages/block-editor/src/components/block-switcher/index.js

```diff
@@ -147,6 +148,9 @@ export const BlockSwitcherDropdownMenu = ( { clientIds, blocks } ) => {
 						} }
 						icon={
 							<>
+								{ isReusable && (
```

Comment:
I think a slot renders nothing if there are no fills, so it should be fine to just render the slot without the surrounding `isReusable` logic.